### ISSUE TITLE
Tacotron2 pretrained model's training parameters

### ISSUE THREAD

**Issue by 2579356425** [espnet/espnet#3293]
Excuse me.What are the training parameters of the Mandarin Tacotron2 pretrained model? I want to use the CSMSC data set to reproduce the pretrained model, but the audio synthesized by the model I trained is different from the pretrained model?

**Comment by sciai-ai** [comment]
I have also encountered the same issue but didnt find any solution yet.

**Comment by kan-bayashi** [comment]
> but the audio synthesized by the model I trained is different from the pretrained model?

Sorry, I could not catch up your meaning?
What kind of difference are there?

**Comment by kan-bayashi** [comment]
You can refer the pretrained models from here:
https://github.com/espnet/espnet/tree/master/egs2/csmsc/tts1#second-results

**Comment by talal-sen** [comment]
in my case the synthesised sound had a background white noise, though I have trained it for 200 operations as in the recipe. Also I noticed that the number of token list of pre-trained model was different from the one in the recipe.

**Comment by sciai-ai** [comment]
in my case the synthesised sound had a background white noise, though I have trained it for 200 operations as in the recipe. Also I noticed that the number of token list of pre-trained model was different from the one in the recipe.

**Comment by kan-bayashi** [comment]
Could you share the genearted sample and ground truth?
I'm wondering the new CSMSC has a difference from the old one.
(I used the open-accessed version)

**Comment by sciai-ai** [comment]
[download (62).zip](https://github.com/espnet/espnet/files/6666987/download.62.zip)

I also used the open access version. 

Token list from pretrained model config:

![image](https://user-images.githubusercontent.com/52277510/122322917-d571aa80-cf58-11eb-9ba5-4b5b33916934.png)

Token list from model trained from espnet2 csmsc recipe 

![image](https://user-images.githubusercontent.com/52277510/122323029-0651df80-cf59-11eb-99b4-845f59ffcb11.png)

**Comment by kan-bayashi** [comment]
Thanks.
Which configuration did you use?
I will run in my environment again.

**Comment by sciai-ai** [comment]
I used the second config as this was generated by the recipe in egs2/csmsc for tacotron

**Comment by 2579356425** [comment]
@kan-bayashi  I preformed ./run.sh which is in the directory espnet/egs2/csmsc/tts1.After the decoding of step 7,I found the synthesized audio in directories /espnet/egs2/csmsc/tts1/exp/tts_train_raw_phn_pypinyin_g2p_phone/decode_train.loss.ave/eval1/wav and /espnet/egs2/csmsc/tts1/exp/tts_train_raw_phn_pypinyin_g2p_phone/decode_train.loss.ave/dev/wav.But the effect of synthesis is not ideal.Here's the synthesized audio. 
[synthesized-audio.zip](https://github.com/espnet/espnet/files/6676973/synthesized-audio.zip)

**Comment by kan-bayashi** [comment]
@2579356425 In your case, the results seem fine because the output in `wav/` is generated with Griffin-Lim.
Please apply the neural vocoder.
https://github.com/kan-bayashi/ParallelWaveGAN#decoding-with-espnet-tts-models-features

You can find the online example of tacotron2 + neural vocoder.
https://colab.research.google.com/github/espnet/notebook/blob/master/espnet2_tts_realtime_demo.ipynb

**Comment by 2579356425** [comment]
@kan-bayashi Thank you.I use the ParallelWaveGAN to synthetic audio and the effect is great.

**Comment by kan-bayashi** [comment]
@sciai-ai I tried the latest master with the latest pypinyin.
But the quality is no problem at all.
[sample.zip](https://github.com/espnet/espnet/files/6683392/sample.zip)
The config is https://github.com/espnet/espnet/blob/master/egs2/csmsc/tts1/conf/tuning/train_tacotron2.yaml

My environment is as follows:
```
$ python check_install.py
[x] python=3.7.3 (default, Mar 27 2019, 22:11:17)  [GCC 7.3.0]

Python modules:
[x] torch=1.5.1
[x] torch cuda=10.1
[x] torch cudnn=7603
[x] torch nccl
[x] chainer=6.0.0
[ ] chainer cuda
[ ] chainer cudnn
[ ] cupy
[ ] torchaudio
[x] torch_optimizer=0.0.1a14
[ ] warpctc_pytorch
[x] warprnnt_pytorch
[ ] warp_rnnt
[ ] chainer_ctc
[ ] pyopenjtalk
[ ] kenlm
[ ] mmseg
[x] espnet=0.9.10
[ ] fairseq
[ ] phonemizer
[ ] gtn

Executables:
[ ] sclite
[ ] sph2pipe
[ ] PESQ
[ ] BeamformIt
```

```
$ pip list | grep pypin
pypinyin               0.42.0
```

Maybe you made something wrong? (e.g. wrong statistics between text2mel and vocoder).
